README.md

```diff
@@ -15,10 +15,10 @@ Our full-stack web development and machine learning curriculum is completely fre
 ## Table of Contents
 
 - [Certifications](#certifications)
-- [The Learning Platform](#the-learning-platform)
+- [The Learning Platform - for users](#the-learning-platform)
```

Comment:
It is definitely possible that developers would want to learn and normal users would want to contribute. Normal users could contribute to documentation and devs are always leveling up their skills.

Reply:
This seems like an unnecessary change, and I agree with Samarmohan.